### PR TITLE
Add CoefficientOfThermalExpansion operators

### PR DESCRIPTION
**Pull request by tm-ren**
Added CTE * TemperatureDelta to CoefficientOfThermalExpansion.extra.cs.
Added TemperatureDelta to TemperatureDelta.extra.cs.
Added corresponding tests to CoefficientOfThermalExpansionTests.cs

### CODE REVIEW

**Comment by tm-ren** [comment]
#1259

File: UnitsNet/CustomCode/Quantities/TemperatureDelta.extra.cs [hunk 1]
```diff
@@ -34,5 +34,11 @@ public partial struct TemperatureDelta
         {
             return Energy.FromJoules(entropy.JoulesPerKelvin * temperatureDelta.Kelvins);
         }
+
+        /// <summary>Get a scalar from a <see cref="TemperatureDelta"/> multiplied by a <see cref="CoefficientOfThermalExpansion"/>.</summary>
+        public static double operator *(TemperatureDelta temperatureDelta, CoefficientOfThermalExpansion cte)
+        {
+            return cte * temperatureDelta;
```
Comment (angularsen) [inline UnitsNet/CustomCode/Quantities/TemperatureDelta.extra.cs:41]: Can you make this an explicit computation with units as you did for CTE?
It reduces indirection and the potential for infinite recursion.

**Review by angularsen — CHANGES_REQUESTED** [review]
Looks good, just a minor request.

File: UnitsNet/CustomCode/Quantities/TemperatureDelta.extra.cs (hunk 1 above)
Comment (tm-ren) [reply UnitsNet/CustomCode/Quantities/TemperatureDelta.extra.cs]: Yes, no problem. Change made.

**Comment by angularsen** [comment]
Thanks! Nuget should be out shortly.

[Release UnitsNet/5.24.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.24.0)

**Pull request merged**